doc/bc/changes-6.0.md

```diff
@@ -158,6 +158,13 @@ Changes affecting version compatibility with former or future versions.
 * `eZ\Publish\API\Repository\Values\ValueObject\SearchHit` has a new property `$matchedTranslation`,
   which will hold language code of the Content translation that matched the search query.
 
+* The `ezrole` table has changed. `version` has been added to the primary key, after the `id`. This is because role
+  drafts are now handled differently. A draft now has the same `id` as the published role, and `version` decides which
+  is which. You should make sure you don't have any role drafts in the database when upgrading. To find drafts, select
+  entries in the `ezrole` table where `version` is not zero. Before this change, the `version` of a draft equals the
+  `id` of the role. The safe way to get rid of stray role drafts is to edit the role in question, and then cancel
+  editing.
```

Comment:
Ok, but I need to put a descriptive text somewhere, since role drafts will not survive the update. Can write it as a comment in an SQL file. So, `data/updates/6.0/mysql.sql` then?